data/processed/industry/employment_culture_entertainment_recreation.yaml

```diff
@@ -0,0 +1,30 @@
+api_doc_url: https://www.nomisweb.co.uk/api/v01/help
+api_type: POST
+data_date: 20200218
+description_short: Employment in cultural, entertainment and leisure industries
+description: This indicator measures level of employment in cultural, entertainment and leisure industries based on the Business Register Employment Survey. Those sectors are identified as clusters of SIC-4 (industry) codes
+endpoint_url: http://www.nomisweb.co.uk/api/v01/dataset/
+framework_group: place_potential
+is_experimental: True
+order: [year, nuts_id, nuts_year_spec, value.id] # USE AS IS: it specifies the order of the exported indicator fields
+region:
+  type: NutsRegion
+  level: 2
+schema:
+  region_id:
+    type: NutsRegion.id
```

Comment:
this should be `type: NutsRegion.nuts_id` (sorry this is being clarified in #120 so you couldn't know)